File: routing-algorithm-python/classes.py
class Agent:
    agentId = 0
    name = ""
    availability = False
    skill = []

    def __init__(self, agentId, name, availability, skill):
        self.agentId = agentId
        self.name = name
        self.availability = availability
        self.skill = skill

    def __repr__(self):
        return f'{self.name}, {self.availability}, {self.skill}'


class Request:
    requestId = 0
    skill = []

    def __init__(self, requestId, skill):
        self.requestId = requestId
        self.skill = skill

    def __repr__(self):
        return f'{self.requestId}, {self.skill}'


class Skill:
    skillId = 0
    name = ""

    def __init__(self, skillId, name):
        self.skillId = skillId
        self.name = name

    def __repr__(self):
        return f'{self.skillId}'

File: routing-algorithm-js/classes.js
//
// This is only a SKELETON file for the 'Hello World' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export { Skill, Request, Agent };
class Skill {
  constructor(skillId, name) {
    this.skillId = skillId;
    this.name = name;
  }
}

class Request {
  constructor(requestId, skill) {
    this.requestId = requestId;
    this.skill = skill;
  }
}

class Agent {
  constructor(agentId, name, availability, skill) {
    this.agentId = agentId;
    this.name = name;
    this.availability = availability;
    this.skill = skill;
  }
}
